cli/tests/e2e/test_output.py

```diff
@@ -92,6 +92,34 @@ def test_output_highlighting__force_color_and_no_color(run_semgrep_in_tmp, snaps
     )
 
 
+# This test is just for making sure that our YAML parser interacts properly
+# with metavariables. We don't want to introduce regressions which might
+# mess this up.
+@pytest.mark.kinda_slow
+def test_yaml_metavariables(run_semgrep_in_tmp, snapshot):
+    stdout, _ = run_semgrep_in_tmp(
+        "rules/yaml_key.yaml",
+        target_name="yaml/target.yaml",
+        output_format=OutputFormat.JSON,
+    )
+    parsed_output = json.loads(stdout)
+    assert "results" in parsed_output
+
+    for result in parsed_output["results"]:
+        value = result["extra"]["metavars"]["$VALUE"]
+        content = value["abstract_content"]
```

Comment:
we should get rid of abstract_content, so there is no need to assert anything on its content.